apps/definition_kafka/lib/kafka/topic/source.ex

```diff
@@ -0,0 +1,99 @@
+defmodule Kafka.Topic.Source do
+  use GenServer, shutdown: 30_000
+  use Annotated.Retry
+  require Logger
+
+  def start_link(t, %Source.Context{} = context) do
```

Comment:
can these be more consistent between source and destination? `t` feels like a generic and confusing in light of the strong shift toward protocols.

Reply:
Just reading through this I'm not sure what t is. Is `t` a `topic`?

Reply:
yeah, I am going to rename those variables